### PR TITLE
Bring update script to the most update state

### PR DESCRIPTION
**Pull request by ruflin**
Remove godep dependencies
Add newest version of config files

### CODE REVIEW

**Comment by monicasarbu** [comment]
LGTM.

File: winlogbeat/make.bat [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-go get github.com/tools/godep
-godep go build
-godep go test -race ./...
-
-REM Coverage report:
-REM godep go test -c -cover -coverpkg ./...
-
-godep go test -c -covermode=atomic -coverpkg ./...
-nosetests -w tests\system --process-timeout=30
+REM Batch script to build and test on Windows. You can use this in conjunction
```
Comment (tsg) [inline winlogbeat/make.bat]: I guess line terminators changed here. @andrewkroh, can this be an issue or it doesn't matter?

**Comment by tsg** [comment]
LGTM, see my question to Andrew above.

In general, I think the updating / generating part can be simplified even more now that the relative paths are known.

**Comment by ruflin** [comment]
@tsg I updated to use relative paths instead of GOPATH.

**Comment by ruflin** [comment]
@andrewkroh It is probably best that you do a brief review (see comment @tsg) and if all ok merge it.

File: packetbeat/pre-commit [hunk 2]
```diff
@@ -2,5 +2,5 @@
 echo "Running pre-commit hook..."
 go fmt ./...
 go vet ./...
-godep go test -short ./...
+go test -short ./...
```
Comment (andrewkroh) [inline packetbeat/pre-commit]: This should set the vendor experiment env variable.

File: winlogbeat/make.bat (hunk 1 above)
Comment (andrewkroh) [inline winlogbeat/make.bat]: If we add '*.bat       eol=crlf' to .gitattributes it should fix this issue.

File: packetbeat/pre-commit (hunk 2 above)
Comment (andrewkroh) [inline packetbeat/pre-commit]: Or maybe just remove the file?

**Comment by andrewkroh** [comment]
Merging to get these changes in. I will open a follow up PR to fix the two issues I brought up.

File: packetbeat/pre-commit (hunk 2 above)
Comment (ruflin) [inline packetbeat/pre-commit]: Lets remove it completely. We would need one on the top level.

File: winlogbeat/make.bat (hunk 1 above)
Comment (ruflin) [inline winlogbeat/make.bat]: @andrewkroh Wasn't aware of this option. Will add it to the file.

**Pull request merged**